src/utils/dom.js

```diff
@@ -17,18 +17,9 @@
 import {assert} from './log';
 import {setStyles} from './style';
 
-/** @const @enum{string} */
-export const styleLinkAttrs = {
-  'rel': 'stylesheet',
-  'type': 'text/css',
-};
-
 /** @const {string} */
 export const styleType = 'text/css';
 
-/** @const {string} */
-export const styleExistsQuerySelector = 'link[rel=stylesheet][href]';
-
```

Comment:
These changes seem unrelated.

Reply:
Oops, I'll create a PR for these changes

Reply:
Here's a PR that remove these extra DOM constants
https://github.com/subscriptions-project/swg-js/pull/2504